src/frontends/tensorflow_lite/tests/convert_model.cpp

```diff
@@ -12,6 +12,9 @@ using TFLiteConvertModelTest = FrontEndConvertModelTest;
 
 static const std::vector<std::string> models{
     std::string("2in_2out/2in_2out.tflite"),
+    std::string("bad_header/zerolen.tflite"),
+    std::string("bad_header/wrong_len_3.tflite"),
+    std::string("bad_header/wrong_pos.tflite"),
```

Comment:
I think you need to create separate test suite `convert_invalid_model` because this one targets normal models